admin/fmt

```diff
@@ -4,7 +4,13 @@ set -xe
 ADMIN=$(dirname "$0")
 PREV=$(pwd)
 if [ ! -d $ADMIN/rustfmt ] ; then
-  git clone --depth 1 https://github.com/rust-lang/rustfmt $ADMIN/rustfmt
+  COMMIT=8829e455b722f1ba8ade915ad2687639a2abb2c3
```

Comment:
Please add a comment above this about why we're pinning the Rustfmt version: We want to ensure that people submitting PRs and CI agree on what version to use, and we want to limit our exposure to breaking changes in the rust-lang/rustfmt repo.

Further, it seems like we should use a fixed version of Rust Nightly, e.g. `+nightly-2021-03-31` to get a consistent build of rustfmt so the script doesn't break when the rust-lang/rust repo is updated.

Further, We need to ensure that that version of Rust Nightly is installed. Note that the runner of this script might not have any version of Rust Nightly installed already, especially if they typically use stable Rust.